README.md

```diff
@@ -1 +1,41 @@
-# hello-microverse
\ No newline at end of file
+
+# Hello microverse
+
+> This is my first project at microverse to practice gihub workflow 
+
+
+
+It just display a simple hello microverse text
```

Comment:
- [ ] [OPTIONAL] For the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description from your README file for the next time 😉).